### PR TITLE
Add a bio for @bf1967

### PR DESCRIPTION
**Pull request by bf1967**
This pull request adds a bio for @bf1967 as requested in #8 

What do you think @PaulSchemenauer?

### CODE REVIEW

File: docs/bf1967.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# BF1967
+
+**Name:** Bob Fox
+
+**Occupation:** Software Developer
+
+**Location:** Margaritaville
```
Comment (christine-erickson) [inline docs/bf1967.md:7]: Bob, can you send me directions to Margaritaville? :tropical_drink:

File: docs/bf1967.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# BF1967
+
+**Name:** Bob Fox
```
Comment (christine-erickson) [inline docs/bf1967.md]: Did you want Bob Fox bold too?

**Comment by christine-erickson** [comment]
Bob - good work, you can commit. :wave:

File: docs/bf1967.md (hunk 1 above)
Comment (MadHatter1) [inline docs/bf1967.md:7]: Me too????

**Pull request merged**